shoop/front/apps/auth/templates/shoop/user/recover_password_mail_content.jinja

```diff
@@ -1,3 +1,7 @@
+Hi {{ user_to_recover.get_full_name() }},
+
+{% trans %}Username{% endtrans %}: {{ user_to_recover.username }}
```

Comment:
This should probably be formatted in a nicer way. Maybe

```
{% trans username=user_to_recover.username %}In case you've forgotten, your username is "{{ username }}".{% endtrans %}
```

somewhere in there?